packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -144,230 +144,292 @@ ECS Container Instance.
 
 %build
 ./scripts/gobuild.sh
-gzip -c scripts/amazon-ecs-init.1 > scripts/amazon-ecs-init.1.gz
 
 %install
-rm -rf $RPM_BUILD_ROOT
-mkdir -p $RPM_BUILD_ROOT/%{init_dir}
-mkdir -p $RPM_BUILD_ROOT/%{bin_dir}
-mkdir -p $RPM_BUILD_ROOT/%{conf_dir}
-mkdir -p $RPM_BUILD_ROOT/%{cache_dir}
-mkdir -p $RPM_BUILD_ROOT/%{data_dir}
-mkdir -p $RPM_BUILD_ROOT/%{dhclient_dir}
-mkdir -p $RPM_BUILD_ROOT/%{man_dir}
-
-install %{SOURCE1} $RPM_BUILD_ROOT/%{init_dir}/ecs.conf
-install %{SOURCE2} $RPM_BUILD_ROOT/%{cache_dir}/ecs-agent-v%{bundled_agent_version}.tar
-install amazon-ecs-init $RPM_BUILD_ROOT/%{bin_dir}/amazon-ecs-init
-install scripts/amazon-ecs-init.1.gz $RPM_BUILD_ROOT/%{man_dir}/amazon-ecs-init.1.gz
-touch $RPM_BUILD_ROOT/%{conf_dir}/ecs.config
-touch $RPM_BUILD_ROOT/%{conf_dir}/ecs.config.json
+install -D amazon-ecs-init %{buildroot}%{_libexecdir}/amazon-ecs-init
+install -D scripts/amazon-ecs-init.1 %{buildroot}%{_mandir}/man1/amazon-ecs-init.1
```

Comment:
We were explicitly compressing this before, but aren't here. Why?

Reply:
https://fedoraproject.org/wiki/Packaging:Guidelines#Manpages

>  When installing man pages, note that they should be installed uncompressed as the build system will compress them as needed.